rlbot_gui/story/story_challenge_setup.py

```diff
@@ -119,6 +120,23 @@ def rlbot_to_player_config(player: dict, team: Team):
     return player_config
 
 
+def script_to_script_config(script: dict):
+    script_path = script["path"]
+    if isinstance(script_path, list):
+        script_path = path.join(*script_path)
+
+    if "$RLBOTPACKROOT" in script_path:
+        for bot_folder in rlbot_gui.bot_folder_settings["folders"].keys():
+            adjusted_folder = path.join(bot_folder, "RLBotPack-master")
+            subbed_path = script_path.replace("$RLBOTPACKROOT", adjusted_folder)
+            if path.exists(subbed_path):
+                print("it exists! (script)")
+                script_path = subbed_path
+                break
```

Comment:
Looks like this code is being duplicated from `rlbot_to_player_config`. Can we instead pull this logic out into a function and use it in both places?